<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>818</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Students Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/icons/window_icon.ico</normaloff>:/resources/icons/window_icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>841</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>200</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Surname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Birthday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NM</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VIPZ</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ukrainian</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>English</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OOP</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>0</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #1565c0;
    border-style: outset;
    border-radius: 4px;
    padding: 5px 18px;
}*/

/*QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #176dce;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/resources/icons/add.ico</normaloff>:/resources/icons/add.ico</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>0</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #e11212;
    border-style: outset;
    border-radius: 4px;
    padding: 5px 18px;
}*/

/*QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #ce1010;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Remove Student</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/resources/icons/remove.ico</normaloff>:/resources/icons/remove.ico</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #1565c0;
    border-style: outset;
    border-radius: 4px;
    padding: 5px 18px;
}
QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #176dce;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Restore list</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/resources/icons/restore.ico</normaloff>:/resources/icons/restore.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
   
 

	/*background: qlineargradient(x1: 0.5, y1: 0.5, x2: 1, y2: 0.5, stop: 0 #ffffff, stop: 1 #d2d2d2);
 background: qlineargradient(x1: 0.5, y1: 0.5, x2: 1, y2: 0.5, stop: 0 #ffffff, stop: 1 #d2d2d2);
border: 1px solid #ababab;
border-radius: 4px;*/
}</string>
    </property>
    <item>
     <property name="text">
      <string>Unspecified</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>[A-Z]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>[Z-A]</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>90</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #1565c0;
    border-style: outset;
    border-radius: 4px;
    padding: 2px 13px;
}

QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #176dce;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/resources/icons/open.ico</normaloff>:/resources/icons/open.ico</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>0</y>
      <width>90</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #1565c0;
    border-style: outset;
    border-radius: 4px;
    padding: 2px 13px;
}

QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #176dce;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/resources/icons/save.ico</normaloff>:/resources/icons/save.ico</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>0</y>
      <width>130</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton
{
    font: 18px 'Arial Rounded MT Bold';
    background-color: white; 
    color: #333333;
    border: 3px solid #292929;
    border-style: outset;
    border-radius: 4px;
    padding: 2px 18px;
}*/

/*QPushButton:pressed
{
    font: bold 17px;
    border: 0px;
}

QPushButton:hover
{
    background-color: #292929;
    color: white;
}

QPushButton:disabled
{
    background-color: #e7e6e6;
    border-color: gray;
}*/</string>
    </property>
    <property name="text">
     <string>Remove All Sudents</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+Del</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>637</x>
      <y>0</y>
      <width>63</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>Sort by:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenu::item:selected
{
	color: #333333;
	background-color: #f0f0f0;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionRemove_all"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/new.ico</normaloff>:/resources/icons/new.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/open.ico</normaloff>:/resources/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/save.ico</normaloff>:/resources/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/saveas.ico</normaloff>:/resources/icons/saveas.ico</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/exit.ico</normaloff>:/resources/icons/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/info.ico</normaloff>:/resources/icons/info.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRemove_all">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/icons/remove.ico</normaloff>:/resources/icons/remove.ico</iconset>
   </property>
   <property name="text">
    <string>Remove all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
